src/main/java/Todo.java

```diff
@@ -0,0 +1,10 @@
+public class Todo extends Task{
```

Comment:
Perhaps consider adding comments before every class declaration such as its inputs, what the object does, what it inherits from, etc :)

Reply:
Ok, I'll consider doing that thanks!